### PR TITLE
Create initial routing links routes

### PR DESCRIPTION
**Pull request by AeonGenesis**
Set up the initial routing for the sidebar navigation.

### CODE REVIEW

**Comment by Riobe** [comment]
You need to merge in master and resolve the conflicts. For instance, you have a change on `Chat` when that component has been removed.

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -40,6 +49,10 @@ const Content = styled.section`
   grid-area: content;
 `;
 
+const Temp = () => {
```
Comment (Riobe) [inline src/components/Layout/Layout.js:52]: I feel like we should remove the `Temp` component and it's route. Right now it's an example for an individual character route, but it's dead code and you have other examples for the actual links. Recreating this with something useful will be part of [Create nav for character](https://trello.com/c/hYqXgwNx/85-create-nav-for-character).

Thoughts?

**Review by Riobe — CHANGES_REQUESTED** [review]
Looks good overall, I just have concerns on the `Temp` component.

File: src/components/Layout/Layout.js (hunk 1 above)
Comment (AeonGenesis) [reply src/components/Layout/Layout.js:52]: Agreed. Doesn't need to be there. I'll remove it.

**Review by Riobe — APPROVED** [review]

**Pull request merged**